cli/src/commands/prev.rs

```diff
@@ -44,6 +44,9 @@ use crate::ui::Ui;
 /// B    B
 /// |    |
 /// A    A
+///
+/// If your working-copy commit is already a non-head commit (i.e. if it has
+/// visible children), then `--edit` is implied.
```

Comment:
This is also bad copypasta, shouldn't it be parents here? 

I assume our users will stumble over this terminology. Can we just simplify it to "If you working-commit is already a commit with a child, then `--edit` is implied".

Reply:
> This is also bad copypasta, shouldn't it be parents here?

No, it's still the same condition for inferring that you want `--edit`.

> I assume our users will stumble over this terminology. Can we just simplify it to "If you working-commit is already a commit with a child, then `--edit` is implied".

True, it's better to use simpler words. Fixed.

Reply:
> > This is also bad copypasta, shouldn't it be parents here?
> 
> No, it's still the same condition for inferring that you want `--edit`.

I understand what you mean in terms of graph terminology. But again, someone who is not aware will expect it to be parents.

Reply:
But it would be incorrect to say parents even if they would expect that :) What do you suggest we write instead?

Reply:
> What do you suggest we write instead?

Not sure. 🤷 I still there is a terminolog missmatch.

Reply:
I'll merge this as is then. If it's confusing users, we will surely hear about it eventually and maybe one of those users have a suggestion.